modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/entry/ListEntries.es.js

```diff
@@ -56,10 +57,14 @@ export default function ListEntries() {
 		languageId: userLanguageId,
 	};
 
+	if (!permissions.view) {
+		return <EmptyStateEntry />;
+	}
+
```

Comment:
@kevenleone, could you apply the same validation on the workflow app ListEntries file?

Reply:
It should work equals on both.